### DISCUSSION TITLE
I want to add another CNAME

### DISCUSSION THREAD

**Discussion by dushmanta05** [is-a-dev/register#7934] [Q&A] [answered]
I'm writing some blogs on **Hashnode**, and I want my blogs to display on my `is-a.dev` main subdomain. I just want to ask if I have to add the CNAME value as a new object in the records and the TXT field value in the `_github_pages_challenges.json` file. Or if it needs some alternative approach, please help me with it. Thanks in advance.

**Comment by wdhdev** [comment]
What are the records Hashnode is asking you to put on your subdomain?

**Reply by dushmanta05** [reply]
Hi! Thanks for your response, the instruction says to add a CNAME Record and TXT Record
![screenshot](https://github.com/is-a-dev/register/assets/73304098/4f2caa2b-f8ee-4be4-8202-15397134ef33)

**Reply by wdhdev** [reply, marked as answer]
We do not support CNAME and TXT records together unfortunately.

**Reply by dushmanta05** [reply]
There's this two files where CNAME & TXT available, so can you confirm again? Thanks.

![Screenshot 1](https://github.com/is-a-dev/register/assets/73304098/81136972-4b93-4f57-ad45-a11fefb66ca9)
![Screenshot 2](https://github.com/is-a-dev/register/assets/73304098/5880fd2b-3a39-449b-8c60-d8d158dd11e4)

**Reply by wdhdev** [reply]
We only support TXT verification for GitHub Pages and Discord as of now. Also, we don't support Vercel either.

**Reply by dushmanta05** [reply]
Okay, thanks for your clarification.
